code/controllers/subsystem/dynamic/dynamic.dm

```diff
@@ -201,6 +201,55 @@ SUBSYSTEM_DEF(dynamic)
 
 	VAR_PRIVATE/next_midround_injection
 
+/proc/get_threat_level_string(shown_threat, fake_greenshift)
```

Comment:
```suggestion
/datum/controller/subsystem/dynamic/proc/get_threat_level_string(shown_threat, fake_greenshift)
```
Like oranges said, as this is only used in the scope of `/datum/controller/subsystem/dynamic` there's no reason for it to be a global proc